data/io.github.fsobolev.Cavalier.gschema.xml

```diff
@@ -53,10 +52,15 @@
 		</key>
 			<key name="line-thickness" type="i">
 			<summary>Thickness of lines</summary>
-			<description>Thickness of lines in "line" mode (in pixels).</description>
+			<description>Thickness of lines non-filled modes (in pixels).</description>
 			<range min="1" max="40"/>
 			<default>15</default>
 		</key>
+		<key name="fill" type="b">
+			<summary>Fill elements</summary>
+			<description>Whether to fill elements in the various modes.</description>
```

Comment:
I'm not sure about name and description. It's not like Wave mode draw _elements_, here's just one continous wave line. I don't have a better ideas for now, but I will think about it.

Reply:
For a name, how about just "Filling"?

Reply:
Good, and for description maybe "Whether shapes should be filled or ourlined". Or "patterns" or "figures" instead of "shapes", not sure what word fits better.

Reply:
I think shapes is fine.